integration/db_integration_test.go

```diff
@@ -1,3 +1,5 @@
+//+build !test_as_root
```

Comment:
@webvictim Any reason you went with build flags instead of checking if the user is root and if they are skipping tests like we have elsewhere?

```
	// This test must be run as root.
	if !isRoot() {
		c.Skip("Tests [...] can only be run as root.")
	}
```

Reply:
I'll see whether I can overhaul this. My intention with the commit that @xacrimon cherry-picked was actually that there would just be one `root_integration_test.go` file where *all* tests that need to run as root would live, to try and avoid too much use of build flags.

Reply:
Ah. IMHO build flags are nice since they allow toggling beyond just being root or not. Say you are root but you don't want to run a bunch of slow tests that snoop around your system files.

Reply:
I agree in principle, but I think the issue is that we (currently) never actually run any of our long-running tests which are gated by flags - only the main test suite gets executed for every PR. We need to improve this situation, but the idea should be to gate as few tests as possible and making build flags more ubiquitous kinda does the opposite.

Reply:
Fair enough. What do you recommend we do?

Reply:
a runtime UID check seems easiest.
build tags are yet another thing we'd need to set to run the tests (in addition to running them as root), so it's a hurdle.

definitely remove the `//+build !test_as_root` tags from all files